src/types/check_config.rs

```diff
@@ -256,4 +274,232 @@ mod tests {
             all_slots.skip(15).step_by(60 * 5).collect::<Vec<usize>>()
         );
     }
+
+    #[test]
+    fn test_should_run() {
+        let tick = Tick::from_time(Utc::now());
```

Comment:
Is time frozen or something, how does this work 🤔

Reply:
Oh it just doesn't matter for the calls that use it, since they never check the tick

Reply:
Ahh I see